RDVFSharp/Commands/Looking/Looking.cs

```diff
@@ -0,0 +1,85 @@
+﻿using FChatSharpLib.Entities.Plugin.Commands;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using System.Threading.Tasks;
+using RDVFSharp.Helpers;
+using System.Timers;
+
+namespace RDVFSharp.Commands
+{
+    public class Looking : BaseCommand<RDVFPlugin>
+    {
+        public Timer LookingTimer = new Timer(7200000);
+        public static Dictionary<string, DateTime> CharacterCooldowns = new Dictionary<string, DateTime>();
+        public static List<string> LookingInformation = new List<string>();
+
+        public class LookingInfo
```

Comment:
Actually, you might want to use this for the Timers as well, so we don't lose track of them.

```
public static List<LookingInfo> LookingInformation = new List<LookingInfo>();

public class LookingInfo
{
            public string CharacterId { get; set; }
            public Timer ExpirationTimer { get; set; }
}
```